Streamlink Twitch GUI/streamlink-twitch-gui.nuspec

```diff
@@ -7,25 +7,59 @@
     <authors>Sebastian Meyer</authors>
     <owners>Sebastian Meyer, Scott Walters</owners>

```

Comment:
@Conjuro Since I'm not the package maintainer, could you please remove my name from the owners field? You should add your name to it. :)
https://docs.microsoft.com/en-us/nuget/reference/nuspec#owners